modules/data-store/src/entity.js

```diff
@@ -0,0 +1,41 @@
+const SerializableClass = require("abitbol-serializable");
+
+/**
+ * A structure that can be added to a ProjectManager.
+ *
+ * @class obsidian-project.lib.structure
+ * @extends abitbol-serializable
+ */
+const Structure = SerializableClass.$extend({
+    __name__: "Structure",
+
+    /**
+     * @property store
+     * @readOnly
+     */
+    getStore() {
+        return this.$data._store; // eslint-disable-line
```

Comment:
I think we should use a `Symbol` here to be sure to not collide with an user-defined property in a sub-class